nautobot-app/{{ cookiecutter.project_slug }}/mkdocs.yml

```diff
@@ -101,6 +101,10 @@ plugins:
 watch:
   - "README.md"
 
+validation:
```

Comment:
If we're going to enforce this across all of the apps, why not use the mkdocs recommendation which is also used by core?

https://www.mkdocs.org/user-guide/configuration/#validation

```yaml
validation:
  omitted_files: warn
  absolute_links: warn
  unrecognized_links: warn
  anchors: warn
```

Reply:
Outside the scope of my concern but no issues adding, is this something that should be added?